client/levels/game_over_screen.tscn

```diff
@@ -9,10 +9,11 @@ script/source = "extends Control
 @export var music: AudioStream
 @export var menu_scene: PackedScene
 
-@onready var _gpu_particles: GPUParticles2D = $GPUParticles2D
+@onready var _gpu_particles: GPUParticles2D = $ParticleContainer/GPUParticles2D
```

Comment:
not an expert, but maybe makes sense to made the GPUParticles2D unique, and then access it via unique with % ?

Reply:
Yeah I think so too.